### PR TITLE
blockdev_inc_backup_without_bitmapmode:Do incremental live backup with     sync:bitmap, but no bitmap-mode

### PR DESCRIPTION
**Pull request by aliang123**
sync:bitmap, but no bitmap-mode

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:1983605

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_without_bitmapmode.q35: PASS (44.47 s)

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks.

File: qemu/tests/blockdev_inc_backup_without_bitmapmode.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        cmd, arguments = inc_backup(self.source_images[0], self.inc_backups[0],
+                                    **extra_options)
+        try:
+            self.main_vm.monitor.cmd(cmd, arguments)
+        except QMPCmdError as e:
+            qmp_error_msg = self.params.get("qmp_error_msg")
+            if qmp_error_msg not in str(e.data):
+                self.test.fail(str(e))
+        else:
+            self.test.fail("Do incremental live  backup without bitmap mode provided")
```
Comment (zhencliu) [inline qemu/tests/blockdev_inc_backup_without_bitmapmode.py:54]: too long line

File: qemu/tests/blockdev_inc_backup_without_bitmapmode.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        3). add target disks for backup to VM via qmp commands
+        4). do full backup and add non-persistent bitmap
+        5). create another file
+        6). do inc bakcup(sync: bitmap and withoout bitmap-mode)
+
+    :param test: test object
+    :param params: test configuration dict
+    :param env: env object
+    """
+    inc_without_bitmapmode = BlockdevIncbkWithoutBitmapmodeTest(test, params, env)
```
Comment (zhencliu) [inline qemu/tests/blockdev_inc_backup_without_bitmapmode.py:78]: too long line

**Comment by zhencliu** [comment]
@aliang123 , it's still about the commit msg, you cut the whole line into two by a new-line char, it appears two lines in the 'git log', but when I patch it, I found the commit msg is one long line, so what about change to the following:

blockdev_inc_backup_without_bitmapmode: negative test
<new-line here>
Do incremental live backup with sync:bitmap, but no bitmap-mode  >>> this is an explanation

File: qemu/tests/blockdev_inc_backup_without_bitmapmode.py (hunk 1 above)
Comment (aliang123) [reply qemu/tests/blockdev_inc_backup_without_bitmapmode.py]: Have updated, thanks zhenchao.

File: qemu/tests/blockdev_inc_backup_without_bitmapmode.py (hunk 2 above)
Comment (aliang123) [reply qemu/tests/blockdev_inc_backup_without_bitmapmode.py]: Have updated, thanks zhenchao.

**Comment by zhencliu** [comment]
LGTM

**Comment by ngu-niny** [comment]
(1/2) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.blockdev_inc_backup_without_bitmapmode:  PASS (102.76 s)
 (2/2) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_inc_backup_without_bitmapmode:  PASS (137.64 s)

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**